packages/datagateway-search/src/table/datasetSearchTable.component.tsx

```diff
@@ -168,7 +168,7 @@ const DatasetSearchTable = (props: DatasetTableProps): React.ReactElement => {
       return linkType === 'investigation'
         ? tableLink(
             `/browse/proposal/${datasetData.investigation.name}/investigation/${datasetData.investigation.id}/dataset`,
-            datasetData.investigation.title
+            datasetData.investigation.name
```

Comment:
Why was this change made? This needs to be switched back as the original was correct (not just for this line, but all lines like this in this file)

Reply:
I just had a look at the master and corrected all of them.

Reply:
I _believe_ this change was actually done by me (but don't quote me on that, check the git blame). My reasoning was that the title was much longer and wordier so didn't fit easily into the table column compared to the name which was always something like `INVESTIGATION 16`. As Louise says, perhaps this wasn't the right idea and wouldn't work well with production data. So I would suggest following her advice.